packages/interactive_media_ads/pigeons/interactive_media_ads_ios.dart

```diff
@@ -275,6 +275,22 @@ enum KeyValueChangeKey {
   unknown,
 }
 
+/// A list of purposes for which an obstruction would be registered as friendly.
+///
+/// See https://developers.google.com/ad-manager/dynamic-ad-insertion/sdk/ios/reference/Enums/IMAFriendlyObstructionPurpose.html.
+enum FriendlyObstructionPurpose {
+  mediaControls,
```

Comment:
Note the IMA docs don't include any doc comments: https://developers.google.com/ad-manager/dynamic-ad-insertion/sdk/ios/reference/Enums/IMAFriendlyObstructionPurpose.html